docs/reference/migration/migrate_8_0/node.asciidoc

```diff
@@ -36,11 +46,18 @@ corresponding node to use this location for its data path. If your nodes each
 have more than one data path in their `path.data` settings then you should move
 all the corresponding subfolders in parallel. Each node uses the same subfolder
 (e.g. `nodes/2`) across all its data paths.
+====
 
-[float]
-==== Rejection of ancient closed indices
-
+.Closed indices created in {es} 7.0 and earlier versions are not supported.
```

Comment:
Should this include v7.0, or should it be prior to 7.0?

Reply:
This should be 6.x. Fixed with bc4b2f39c9b. Thanks!